runtime/opaque/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/runtime/opaque/opaque_cpp.cpp
```

Comment:
this cmake file seems suspect. this is based on a C++ file, but you have an ll file here. I suspect a copy paste error. You should take a closer look.